src/database/migrations/V1__create_jobs_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS jobs (
+    id TEXT PRIMARY KEY NOT NULL,
+    name TEXT NOT NULL UNIQUE,
+    status TEXT NOT NULL,
+    project TEXT NOT NULL,
+    commands TEXT NOT NULL,
+    started_at TEXT NOT NULL,
+    ended_at TEXT
```

Comment:
[PP] We should use timestamp instead of text. And we can default to `now()` as well as use `now()` when updating `ended_at` on job end.


```suggestion
    started_at TIMESTAMP NOT NULL DEFAULT now(),
    ended_at TIMESTAMP
```

Reply:
Yeah I think we need to investigate a bit, let's iterate in another PR.